### PR TITLE
Fix postgres double quote issue.  

### PR DESCRIPTION
**Pull request by granadacoder**
Fix for:

https://github.com/thepirat000/Audit.NET/issues/623

### CODE REVIEW

File: src/Audit.NET.PostgreSql/Providers/PostgreSqlDataProvider.cs [hunk 1]
```diff
@@ -248,7 +248,7 @@ public IEnumerable<AuditEvent> EnumerateEvents(string whereExpression)
         public IEnumerable<T> EnumerateEvents<T>(string whereExpression) where T : AuditEvent
         {
             var where = string.IsNullOrWhiteSpace(whereExpression) ? "" : $"WHERE {whereExpression}";
-            var sql = $@"SELECT ""{GetDataColumnName(null)}"" FROM {GetFullTableName(null)}"" {where}";
+            var sql = $@"SELECT ""{GetDataColumnName(null)}"" FROM {GetFullTableName(null)} {where}";
             return EnumerateEventsByFullSql<T>(sql);
         }
```
Comment (granadacoder) [inline src/Audit.NET.PostgreSql/Providers/PostgreSqlDataProvider.cs:253]: The 
 
    GenerateFullTableName(null)

produces

    "myschema"."MyAuditEventTable"

(it is already double-quote delimited)

Thus the extra double quote (before the WHERE) is unnecessary and causing a bug.

Comment (thepirat000) [reply src/Audit.NET.PostgreSql/Providers/PostgreSqlDataProvider.cs:253]: Thanks! I will add some unit tests and fix the same for the other overload

**Comment by granadacoder** [comment]
Hey.  You closed this PR?  I guess to "super fix it" (on your end) in a different PR?

Friendly FYI:
This is actually a blocking issue for us.  we "need" to upgrade to (beyond 20.1.6), but this is breaking us.

**Comment by thepirat000** [comment]
that's correct, I've fixed the issue in a different PR, also added unit tests to the EnumerateEvents methods

**Pull request closed without merging**